sdk/core/azure-core-tracing-opentelemetry/src/samples/AsyncListKeyVaultSecretsSample.md

```diff
@@ -18,97 +18,83 @@ Sample uses **[opentelemetry-sdk][opentelemetry_sdk]** as implementation package
 </dependency>
 ```
 
-[//]: # ({x-version-update-start;com.azure:azure-security-keyvault-secrets;current})
 ```xml
+<!-- SDK dependencies   -->
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-identity</artifactId>
+    <version>1.0.5</version>
+</dependency>
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-secrets</artifactId>
     <version>4.2.0-beta.1</version>
 </dependency>
-```
-[//]: # ({x-version-update-end})
-[//]: # ({x-version-update-start;com.azure:azure-core-tracing-opentelemetry;current})
-```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-core-tracing-opentelemetry</artifactId>
     <version>1.0.0-beta.4</version>
 </dependency>
 ```
-[//]: # ({x-version-update-end})
 
 #### Sample demonstrates tracing when asynchronously creating and listing secrets from a Key Vault using [azure-security-keyvault-secrets][azure_keyvault_secrets] client library.
 ```java
```

Comment:
We should update this Sample to use an inject snippet, should be able to include an entire file in a snippet

Reply:
Yup, planning to do that in #10321 and use these sample snippets in the Readme content.